addons/website_profile/controllers/main.py

```diff
@@ -37,13 +37,17 @@ def _check_avatar_access(self, user_id, **post):
 
     def _check_user_profile_access(self, user_id):
```

Comment:
the previous implementation return False instead of `user_sudo` if we can't access the user, can we keep that logic ?
so you return False as user or as denial_reason everytime